sonatype-idea.xml

```diff
@@ -104,6 +104,9 @@
   <JSCodeStyleSettings>
     <option name="SPACE_BEFORE_FUNCTION_LEFT_PARENTH" value="false" />
   </JSCodeStyleSettings>
+  <TypeScriptCodeStyleSettings version="0">
+    <option name="SPACES_WITHIN_OBJECT_TYPE_BRACES" value="false" />
+  </TypeScriptCodeStyleSettings>
```

Comment:
this and a few other rules actually appear to be deviations from our JS rules that please the [google typescript style guide](https://github.com/google/gts), I'll verify if that's what we really want (i think it is?)